src/data/eventSchedule.js

```diff
@@ -109,45 +124,57 @@ const eventSchedule = [
 			}
 		]
 	},
+    {
+		startTime: getDateObj('05', '19:00:00'),
+		events: [
+			{
+				name: 'Meme Compeition Ends',
+				duration: '',
+				location: 'Discord: #memes',
+				description: `Submit your last memes to the #memes channel on Discord! 
+                The winner will earn extra raffle entries ;)`
+			}
+		]
+	},
 	{
-		startTime: getDateObj('26', '21:00:00'),
+		startTime: getDateObj('05', '21:00:00'),
 		events: [
 			{
 				name: 'Hacking Ends',
 				duration: '',
 				location: '',
 				description: `Your amazing project should be done and submitted to
-				Devpost for judging. Also remember to submit the 'Team Confirmation'
-				google form.`
+				Devpost for judging. Also remember to submit the 'HOTH X Team Submission'
+				Google form.`
 			}
 		]
 	},
 	{
-		startTime: getDateObj('26', '21:15:00'),
+		startTime: getDateObj('05', '21:15:00'),
 		events: [
 			{
 				name: 'Judging',
 				duration: '75 min',
 				location: '',
-				description: `Our judges will look through your devpost submissions.
+				description: `Our judges will look through your Devpost submissions.
 				Take this time to relax and refresh before the closing ceremony.`
 			}
 		]
 	},
 	{
-		startTime: getDateObj('26', '22:30:00'),
+		startTime: getDateObj('05', '22:30:00'),
 		events: [
 			{
 				name: 'Closing Ceremony',
 				duration: '20 min',
 				location: 'Covel Grand Horizon/Zoom',
```

Comment:
Just want to make sure that we are actually having the closing ceremony on zoom as well